packages/core/src/lib/wait_for_remote_peer.ts

```diff
@@ -61,11 +62,11 @@ export async function waitForRemotePeer(
   }
 
   if (protocols.includes(Protocols.PeerExchange)) {
```

Comment:
We do not expect users to use peer exchange directly but only as a libp2p peer discovery protocol. Hence peer exchange support can be dropped from `waitForRemotePeer` and also potentially from the `Protocols` enum.